networking/kvm.go

```
@@ -478,7 +478,7 @@ func kvmSetup(podRoot string, podID types.UUID, fps []ForwardedPort, netList com
 			}
 			err = netlink.LinkSetMaster(link, br)
 			if err != nil {
-				// TODO: cleanup tap interface
+				tuntap.RemovePersistentIface(n.runtime.IfName, tuntap.Tap)
```

Even if we ignore the error, should we print a warning with it if this fails?